governance/shutdown.go

```diff
@@ -22,7 +22,6 @@ import (
 // There are other workers responsible for other functions, which will also so some cleanup when the Node Shutdown Message
 // arrives. For example, the node heartbeat function is stopped by the Agreement worker.
 func (w *GovernanceWorker) nodeShutdown(cmd *NodeShutdownCommand) {
-	glog.V(3).Infof(logString(fmt.Sprintf("begin node shutdown process.")))
```

Comment:
It is really important for this debug message to remain here. Moving it down isnt helpful. I know we will get 2 shutdown begin msgs but they are slightly different, I think that's good because its clear what is happening.

Reply:
ok, ok, moved it up there.